addons/website_event_track/views/event_track_templates_list.xml

```diff
@@ -122,18 +124,18 @@
                         <div t-attf-class="o_wevent_offcanvas_tag_#{tag_category.name} accordion-collapse collapse">
                             <div class="accordion-body pt-0">
                                 <ul class="list-group list-group-flush">
-                                        <li t-if="tag_category.tag_ids and any(tag.color for tag in tag_category.tag_ids)" class="list-group-item">
-                                            <t t-foreach="tag_category.tag_ids" t-as="tag">
+                                    <t t-if="tag_category.tag_ids and any(tag.color for tag in tag_category.tag_ids)" t-foreach="tag_category.tag_ids" t-as="tag">
```

Comment:
This change is annoying but indeed, it was badly written ^^ ... well let us hope people did not make too strict xpath on this one :sweat_smile: